### PR TITLE
potential fix for #398

### PR DESCRIPTION
**Pull request by mr-tz**

### CODE REVIEW

File: capa/rules.py [hunk 1]
```diff
@@ -866,7 +866,8 @@ def _get_rules_for_scope(rules, scope):
         given a collection of rules, collect the rules that are needed at the given scope.
         these rules are ordered topologically.
 
-        don't include "lib" rules, unless they are dependencies of other rules.
+        don't include auto-generated "subscope"/"lib" rules.
```
Comment (williballenthin) [inline capa/rules.py:869]:
```suggestion
        don't include auto-generated "subscope" rules.
```

**Review by williballenthin — CHANGES_REQUESTED** [review]
needs update to test `test_lib_rules`, then good to go

File: capa/rules.py (hunk 1 above)
Comment (williballenthin) [reply capa/rules.py:869]: oh man i like this suggestion feature

Comment (mr-tz) [reply capa/rules.py:869]: Yes, very cool! Thanks for the feedback.

**Pull request merged**